Changelog.md

```diff
@@ -5,6 +5,7 @@
 ## Unreleased
 
 - Swift: Do not stop analysis if we encounter a badly formatted project.pbxproj file ([#1177](https://github.com/fossas/fossa-cli/pull/1177))
+- Adds `--json` flag to `fossa container analyze`
```

Comment:
This Unreleased section got turned into a release on master, so tou'll need to rebase onto master or merge master in and put this in an `Unreleased` section by itself.